src/V3AstNodeOther.h

```diff
@@ -2276,6 +2276,7 @@ class AstTask final : public AstNodeFTask {
 class AstCFile final : public AstNodeFile {
     // C++ output file
     // Parents:  NETLIST
+    int64_t m_complexityScore = 0;
```

Comment:
Why can't we use the existing splitSize metrics?  I'd prefer no new accounting systems...

And if you can't use the existing counter, at least increment using same metrics, e.g. instrCount.

Reply:
Switched to splitSize metrics.

BTW, splitSize is a twin feature, so I think another nice name for the option would be `--output-join`, and it could probably also be size-based instead of file-count-based then.  Combining the two would make sense if there were both gigantic .cpp files (to be split) and a lot of small files (to be joined).

Reply:
That naming probably makes sense.  However I suspect people would want to pick the output as a fixed number based on the number of cores?  Or maybe join to that number of files, but more if exceeds some maximum number of statements?  How does this relate to your use case?

Reply:
File count is fine I think, but I am not sure if any fixed default value would make sense.  Currently size of output files is bounded by default, and with such a fixed value the output files could grow endlessly with big designs.  I believe C++ compilers do not generally cope well with it.